frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -81,7 +81,13 @@ export const AppVersionsManager = function AppVersionsManager({
 
   const createVersion = (versionName, createAppVersionFrom) => {
     versionName = versionName.trim();
-    if (versionName !== '') {
+    if (versionName.length > 25) {
+      toast.error('The version name should not be longer than 25 characters');
+      setIsCreatingVersion(false);
+    } else if (versionName == '') {
+      toast.error('The version name should not be empty');
+      setIsCreatingVersion(false);
+    } else {
```

Comment:
```suggestion
    if (versionName.length > 25) {
      toast.error('The version name should not be longer than 25 characters');
      setIsCreatingVersion(false);
      return;
    }
    if (versionName == '') {
      toast.error('The version name should not be empty');
      setIsCreatingVersion(false);
      return;
    }
```
To get rid of if else